### PR TITLE
The post before JB will be here.

### PR DESCRIPTION
**Pull request by wolframkriesing**

### CODE REVIEW

File: _posts/2017-10-13-jb-in-da-house.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+feature_image: posts/2017-10-13-jbhc17/poster.jpg
+---
+
+It started in May, that we thought about inviting [J.B. Rainsberger][jbrains] to HolidayCheck. Yeah, there was some bias from my side, since I had the pleasure to have learned from him a couple years ago. So I was all in. I just had to convince my colleagues and the company to pay for it. And it went quite smooth. HolidayCheck is quite committed to improve continuously.
+
+[jbrains]: https://twitter.com/jbrains
+
+## 7:26min
+
+If you have never heard of “7 minutes 26 seconds”? Now you did! It is a short and inspiring [lightning talk][7-26-talk] by J.B. called “7 minutes, 26 seconds, and the Fundamental Theorem of Agile Software Development”. Here at HolidayCheck people refer to it as the “Accidental Complication“ talk. I think the talk itself inspired many people here to be as excited about having J.B. around.
```
Comment (smolak) [inline _posts/2017-10-13-jb-in-da-house.md]: Talk is being mentioned probably too many times. Perhaps if you skip `talk` after `Accidental Complication` it will look better.

Comment (wolframkriesing) [reply _posts/2017-10-13-jb-in-da-house.md]: done, thx

File: _posts/2017-10-13-jb-in-da-house.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+[7-26-talk]: https://vimeo.com/79106557
+[jbhc17]: https://twitter.com/search?q=%23jbhc17&src=typd
+
+## Remove bottlenecks
+
+The topic “Move the bottleneck out of product development” makes it very clear that our goal is to lead the market. Once we have no bottlenecks in product development. Once we deliver the best thing for our Urlauber. Once we run at high speed. Once we have no useless friction slowing us down, we will be able to lead the market. We will deliver the best value. And we will have even more fun and be the best Urlauber company. And we get where we want to be: we will have to look for the bottlenecks in the market, not in our company anymore. We will be able to try new business models, we will try to innovate, build, measure and learn at high speed.
```
Comment (smolak) [inline _posts/2017-10-13-jb-in-da-house.md]: `Once we ran out of speed.` (mind the dot at the end) sounds like we did run out of speed at some point in time. How about splitting those 4 sentences with a `,` creating list of things to be done (dependencies) and finally `we will be able to ...`?

But I might be mistaken and this perhaps is a proper way of listing those things out.

Comment (smolak) [reply _posts/2017-10-13-jb-in-da-house.md]: As you use `will`, I guess you should add it here as well: `And we *will* get where we want`.

Comment (smolak) [reply _posts/2017-10-13-jb-in-da-house.md]: This whole sentence (`And we get where we want to be: we will have to look for the bottlenecks in the market, not in our company anymore.`) sounds a bit odd:
 - will we **have to** look for bottlenecks in the market? I'd rather look for wide open highways, opportunities :) (or at things like [Kim is doing](http://kimjongunlookingatthings.tumblr.com/) ;) )

The next sentence fits very well - `we will be able to try new business models`.

File: _posts/2017-10-13-jb-in-da-house.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+That is why we invited J.B. to learn where we can improve in those areas.
```
Comment (smolak) [inline _posts/2017-10-13-jb-in-da-house.md]: You need at least a `,` or `.` or another conjunction to separate what we did and why, because right now it reads that we invited JB for **him** to learn where we can improve, and not JB to **teach us** where we can improve.

File: _posts/2017-10-13-jb-in-da-house.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+## How?
+
+On Monday we will start with a keynote, have a nice lunch and will then be able to pick the right first bottleneck per team and practice the tools to fix it. J.B. will be leading us and be available to help, teach and practice with us.
```
Comment (smolak) [inline _posts/2017-10-13-jb-in-da-house.md]: `teaching and practicing with us`

File: _posts/2017-10-13-jb-in-da-house.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+On Tuesday we will do a full-time workshop, continue applying the learnings and get a deeper insight on “Value-Driven Product Development”.
+
+## You would also like to?
```
Comment (smolak) [inline _posts/2017-10-13-jb-in-da-house.md]: `Would you like to do it as well?` <-- is that what you meant?

File: _posts/2017-10-13-jb-in-da-house.md (hunk 2 above)
Comment (wolframkriesing) [reply _posts/2017-10-13-jb-in-da-house.md]: @smolak hehe, this is actually a reference to the book project phoenix, which mentions this. once you have no bottlenecks left in the company because u can move at lightning speed, there is no bottleneck in the company anymore. So it must be external. Which is the market and we, in the end, want to determine and lead the market.

File: _posts/2017-10-13-jb-in-da-house.md (hunk 3 above)
Comment (wolframkriesing) [reply _posts/2017-10-13-jb-in-da-house.md]: :) thx

File: _posts/2017-10-13-jb-in-da-house.md [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+
+Well, it is quite an awesome thing to take part in such an event. And it is cool to learn those things on the job. And you can, by being part of HolidayCheck. Want to [apply]?
```
Comment (smolak) [inline _posts/2017-10-13-jb-in-da-house.md]: Perhaps it will be better if you will start with `It is an awesome`, skipping the well, quite. You will sound more confident and show how cool this event is.

File: _posts/2017-10-13-jb-in-da-house.md (hunk 5 above)
Comment (wolframkriesing) [reply _posts/2017-10-13-jb-in-da-house.md]: i want to say "you also like to learn from JB?" => join HC :)

File: _posts/2017-10-13-jb-in-da-house.md (hunk 6 above)
Comment (wolframkriesing) [reply _posts/2017-10-13-jb-in-da-house.md]: +1

Comment (smolak) [reply _posts/2017-10-13-jb-in-da-house.md]: `And what is even better, we do it during working hours. And so can you, by being part of HolidayCheck.` <-- what do you think?

Comment (wolframkriesing) [reply _posts/2017-10-13-jb-in-da-house.md]: mmmh, its both during working hours and on the job. i would leave it

File: _posts/2017-10-13-jb-in-da-house.md (hunk 5 above)
Comment (smolak) [reply _posts/2017-10-13-jb-in-da-house.md]: Then I would rephrase it to what I suggested. Because yours lacks verb, e.g. 'doing' hence 'do it'. I am not expert in english, but I guess it will sound better with than without it.

File: _posts/2017-10-13-jb-in-da-house.md (hunk 6 above)
Comment (smolak) [reply _posts/2017-10-13-jb-in-da-house.md]: OK.

File: _posts/2017-10-13-jb-in-da-house.md (hunk 2 above)
Comment (smolak) [reply _posts/2017-10-13-jb-in-da-house.md]: OK :)
